packages/@glimmer/integration-tests/test/strict-mode-test.ts

```diff
@@ -0,0 +1,723 @@
+import {
+  RenderTest,
+  test,
+  jitSuite,
+  GlimmerishComponent,
+  trackedObj,
+  defineComponent,
+  defineSimpleHelper,
+  defineSimpleModifier,
+} from '..';
+
+class GeneralStrictModeTest extends RenderTest {
+  static suiteName = 'strict mode: general properties';
+
+  // @test
+  // 'Passing helpers as arguments to components'() {
+  //   const plusOne = defineSimpleHelper((value = 0) => value + 1);
+  //   const Foo = defineComponent({}, '{{@asHelper 123}} {{@asValue}} {{@asValueWithArg}}');
+  //   const Bar = defineComponent(
+  //     { plusOne, Foo },
+  //     '<Foo @asHelper={{plusOne}} @asValue={{(plusOne)}} @asValueWithArg={{plusOne 456}}/>'
+  //   );
+
+  //   this.renderComponent(Bar);
+  //   this.assertHTML('124 1 457');
+  //   this.assertStableRerender();
+  // }
```

Comment:
Why is this commented out?

Reply:
These tests are tests that I've written and should be included in the near future (e.g. next PR or two) but that I didn't want to include in this PR. I can keep them locally instead, wasn't sure which would be better to do. I _think_ there's a `@skip` potentially also, but tbh I think comments are basically equivalent to that.

Reply:
Ya, I'd prefer to break these out or @skip, but I don't feel super strongly.

Reply:
converted to use `skip: true`, and made that work